scripts/create_venv.sh

```diff
@@ -5,7 +5,7 @@ set -e
 cur_dir=$(dirname ${BASH_SOURCE[0]})
 source $cur_dir/helpers.sh
 
-/usr/local/bin/python3 -m venv venv
+python3 -m venv venv
```

Comment:
We have the option to keep this as 
```suggestion
/usr/bin/env python3 -m venv venv
```
But I think it would be valuable to if we add a dep on `pyenv` in order to have the expected python version, however maybe this library should be agnostic about the python version (^3.6)